crates/bevy_winit/src/accessibility.rs

```diff
@@ -28,7 +28,7 @@ use bevy_window::{PrimaryWindow, Window, WindowClosed};
 #[derive(Default, Deref, DerefMut)]
 pub struct AccessKitAdapters(pub EntityHashMap<Adapter>);
 
-/// Maps window entities to their respective [`WinitActionRequests`]s.
+/// Maps window entities to their respective [`ActionRequest`]s.
```

Comment:
`WinitActionRequests` doesn't seem to exist, the type that seemed to match this description was `ActionRequest`, but I'm not sure it's correct.